peekingduck/pipeline/nodes/input/utils/read.py

```diff
@@ -173,18 +182,26 @@ class VideoNoThread:
     """
 
     def __init__(self, input_source: str, mirror_image: bool) -> None:
+        self.logger = logging.getLogger("VideoNoThread")
```

Comment:
Consider `type(self).__name__` so it will print what we want even when the class name gets changed. (Similar for `VideoThread` as well).

Also, in the logs, this will show only `VideoNoThread` as the logger name without the `peekingduck.pipeline...` prefix.

Since we are printing info logs, i.e., this will be show to users by default, will it be better to prepend the `peekingduck.pipeline...` prefix so the logs looks consistent?

Also, should `f"read_frame: ret={ret}, #frames read={self._frame_counter}"` really be an info log instead of a debug log? The logging message looks kind of cryptic and more like debugging logs to me.